internal/command/ls_runs.go

```diff
@@ -37,11 +40,13 @@ func init() {
 type lsRunsCmd struct {
 	cobra.Command
 
-	csv    bool
-	after  flag.DateTimeFlagValue
-	before flag.DateTimeFlagValue
-	sort   *flag.Sort
-	quiet  bool
+	csv      bool
+	after    flag.DateTimeFlagValue
+	before   flag.DateTimeFlagValue
+	inputStr string
+	sort     *flag.Sort
+	limit    int
```

Comment:
```suggestion
	limit    uint
```

This would ensure that you can not pass invalid negative values on the command line for `--limit`